packages/patternfly-4/react-charts/src/components/ChartBullet/utils/chart-bullet-theme.ts

```diff
@@ -27,9 +27,9 @@ interface ChartBulletThemeInterface {
 const getLegendColorScale = (computedData: any, legendData: any) => {
   const colorScale: string[] = [];
   legendData.forEach((data: any, index: number) => {
-    for (let i = 0; i < computedData.length; i++) {
-      if (index === computedData[i]._index) {
-        colorScale.push(computedData[i]._color);
+    for (const i of computedData) {
+      if (index === i._index) {
```

Comment:
Where are you getting the `index` from? Wouldn't this have to be something like `for (const [i, v] of`?

Reply:
`index` comes from the `forEach` loop on line 29, and wasn't touched as part of this change.  This rule was new to me, you can find more on it here: [prefer-for-of](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-for-of.md)